scripts/install-deps.sh

```diff
@@ -156,6 +159,25 @@ do_install_githooks() {
     fi
 }
 
+do_install_openssl() {
+    local openssl_dir=$work_folder/openssl-1.1.1f
+    echo "Installing OpenSSL ..."
+
+    $SUDO apt-get install -y build-essential checkinstall zlib1g-dev -y || return
+    wget https://www.openssl.org/source/openssl-1.1.1f.tar.gz || return
+    mkdir -p $openssl_dir || return
+    tar -xf openssl-1.1.1f.tar.gz -C $openssl_dir --strip-components=1 || return
+    pushd $openssl_dir > /dev/null || return
+    ./config --prefix="$openssl_dir_path" --openssldir="$openssl_dir_path" shared zlib || return
```

Comment:
openssl_dir_path is based on /tmp workfolder, so the shared .so lib files will be in /tmp which is typically mounting a tmpfs filesystem (runs only in RAM).  After a reboot, these .so library will be deleted.  Therefore, I recommend adding `--openssl-dir-path` argument to install-deps.sh to allow script user to specify a more permanent location to save the openssl 1.x .so files.